### PR TITLE
update references to editions and turn change them to refer to plans

### PR DESCRIPTION
**Pull request by mooreds**
We made this change in our marketing long ago, but it never got pushed through our docs and blog.

Also added a check to make sure we don't use edition going forward.

### CODE REVIEW

File: astro/src/content/docs/get-started/run-in-the-cloud/cloud.mdx [hunk 1]
```diff
@@ -628,10 +628,10 @@ A support request is required because data exports contain sensitive fields, lik
 
 If you need to download user data regularly for analytics or other purposes, consider using the [User API](/docs/apis/users) or a [webhook](/docs/extend/events-and-webhooks/) to track ongoing account creation. Be aware that performing such operations as downloading all users in your account can impact performance; if possible, pull a subset of users, such as those who have been created in the last week.
 
-If these solutions do not meet your needs and you have an Enterprise edition, open a [support ticket](https://account.fusionauth.io/account/support/) to discuss options for regular data exports.
+If these solutions do not meet your needs and you have an Enterprise plan, open a [support ticket](https://account.fusionauth.io/account/support/) to discuss options for regular data exports.
```
Comment (mark-robustelli) [inline astro/src/content/docs/get-started/run-in-the-cloud/cloud.mdx:631]: Is this Enterprise plan or Enterprise Plan?

File: astro/src/content/docs/operate/roadmap/roadmap.mdx [hunk 2]
```diff
@@ -54,7 +54,7 @@ Some functionality, however, is more liable to change. See <ScrollRef target="Te
 
 Sometimes you just can't wait for a feature to be finished through the normal development process! We get it. 
 
-If you'd like to pay us to build a feature, please [contact us](/contact) and ask about professional services. Please note that such a contract usually requires an Enterprise edition contract.
+If you'd like to pay us to build a feature, please [contact us](/contact) and ask about professional services. Please note that such a contract usually requires an Enterprise plan with a contract.
```
Comment (mark-robustelli) [inline astro/src/content/docs/operate/roadmap/roadmap.mdx:57]: need to make consistent with [Name] [Pp]lan. Across changes.

**Review by mark-robustelli — CHANGES_REQUESTED** [review]
Overall looks good.  A few comments.  Most are around how we use "Community plan" vs "Community Plan".  I saw the same thing came up with Enterprise [Pp]lan.

File: astro/src/content/blog/announcing-fusionauth-1-52.mdx [hunk 3]
```diff
@@ -1,7 +1,7 @@
 ---
 publish_date: 2024-08-06
 title: Announcing FusionAuth 1.52.0 - Passkeys for Everyone!
-description: This update brings passkeys to the free Community Edition of FusionAuth, enhancing security for everyone.
+description: This update brings passkeys to the free Community Plan of FusionAuth, enhancing security for everyone.
```
[the remark this answers is not in the document]
Comment (mooreds) [reply astro/src/content/blog/announcing-fusionauth-1-52.mdx:4]: Thanks, we should standardize.

File: astro/src/content/blog/fusionauth-funding.mdx [hunk 4]
```diff
@@ -14,7 +14,7 @@ Being a bootstrapped company has always been a source of pride for us. However,
...
 
-What sets FusionAuth apart is our [free community edition](/download/?utm_source=blog&utm_medium=blog+post&utm_campaign=funding-announcement), which has been downloaded over 13 million times. This edition empowers developers to seamlessly integrate registration, login, and user management features into their applications while adhering to industry standards.
+What sets FusionAuth apart is our [free community option](/download/?utm_source=blog&utm_medium=blog+post&utm_campaign=funding-announcement), which has been downloaded over 13 million times. This plan empowers developers to seamlessly integrate registration, login, and user management features into their applications while adhering to industry standards.
```
[the remark this answers is not in the document]
Comment (mooreds) [reply astro/src/content/blog/fusionauth-funding.mdx:17]: Yeah, I thought linking to the download page and calling it a plan was a bit weird, but maybe I'm wrong.

File: astro/src/content/blog/announcing-fusionauth-1-52.mdx (hunk 3 above)
[the remark this answers is not in the document]
Comment (mooreds) [reply astro/src/content/blog/announcing-fusionauth-1-52.mdx]: I standardized on lower case P, upper case plan name. So like 'Enterprise plan'. This seemed to be the most common.

**Review by mark-robustelli — APPROVED** [review]
LGTM

**Review by lyleschemmerling — APPROVED** [review]

**Pull request merged**